todo_app/app.py

```diff
@@ -3,47 +3,70 @@
 from todo_app.flask_config import Config
 from todo_app.data.session_items import get_items, get_item, add_item
 from todo_app.data.trello_items import TrelloService
+from todo_app.data.view_model import ViewModel
 
-app = Flask(__name__)
-app.config.from_object(Config())
-
-# Create an instance of the TrelloService class with the API key and API token
-api_key = os.getenv('API_KEY')
-api_token = os.getenv('API_TOKEN')
-trello_service = TrelloService(api_key, api_token)
-
-# Update the function that handles the GET request for fetching todo items
-@app.route('/')
-def index():
-    # Get Board ID and List ID from env file
-    list_id = os.getenv('TO_DO_LIST_ID')
-    # Use the TrelloService class method to fetch todo items
-    todo_items = trello_service.fetch_todo_items(list_id)
-    
-    return render_template('index.html', items=todo_items)
-
-@app.route('/add', methods=['POST'])
-def add_new_card():
-    # Get List ID from env file
-    list_id = os.getenv('TO_DO_LIST_ID')
-    card_name = request.form.get('item')
-    updated_items = add_item(card_name)
-    # Call the TrelloService class method to create a new card
-    card_id = trello_service.create_todo_card(list_id, card_name)
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+
+
+    # Create an instance of the TrelloService class with the API key and API token
+    api_key = os.getenv('API_KEY')
+    api_token = os.getenv('API_TOKEN')
+    trello_service = TrelloService(api_key, api_token)
+
+    # Update the function that handles the GET request for fetching todo items
+    @app.route('/')
+    def index():
+        # Get Board ID and List ID from env file
+        to_do_list_id = os.getenv('TO_DO_LIST_ID')
+        doing_list_id = os.getenv('DOING_LIST_ID')
+        done_list_id = os.getenv('DONE_LIST_ID')
+        
+        # Use the TrelloService class method to fetch items by status
+        todo_items = trello_service.fetch_todo_items(to_do_list_id)
+        doing_items = trello_service.fetch_todo_items(doing_list_id)
+        done_items = trello_service.fetch_todo_items(done_list_id)
```

Comment:
If you look at these items that get returned, you'll see that they don't actually have their statuses set as you'd expect - they literally just have "list[name]" - this is because you're passing through the `list` _class_ rather than an actual Trello list in the `fetch_todo_items` function (line 28) - instead given your structure we probably want:
* To pass the list name through directly, so we need to change the Item class to expect that:
```python
    @classmethod
    def from_trello_card(cls, card, list_name):
        return cls(card['id'], card['name'], list_name)
```
* To pass that into the `fetch_todo_items` function so it knows what to pass on:
```python
def fetch_todo_items(self, list_id, list_name):
```
* And then to update so that both here in app.py where we call that function, and where you call `Item.from_trello_card` is called the list_name also gets passed